modules/client/examples.md

```diff
@@ -1229,27 +1238,31 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Create an address list client
 const client = new ClientAddressList(contextPlugin);
 
-const proposalParams: ICreateProposalParams = {
-  pluginAddress: "0x1234567890123456789012345678901234567890",
-  metadata: {
-    title: "Test Proposal",
-    summary: "This is a short description",
-    description: "This is a long descrioption",
-    resources: [
-      {
-        name: "Discord",
-        url: "https://discord.com/...",
-      },
-      {
-        name: "Website",
-        url: "https://website...",
-      },
-    ],
-    media: {
-      logo: "https://...",
-      header: "https://...",
+const metadata: ProposalMetadata = {
+  title: "Test Proposal",
+  summary: "This is a short description",
+  description: "This is a long description",
+  resources: [
+    {
+      name: "Discord",
+      url: "https://discord.com/...",
+    },
+    {
+      name: "Website",
+      url: "https://website...",
     },
+  ],
+  media: {
+    logo: "https://...",
+    header: "https://...",
   },
+};
+
+const cid = await client.methods.pinMetadata(metadata);
+
+const proposalParams: ICreateProposalParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+  metadata: cid,
```

Comment:
Should `metadata` now be called something like `metadataUri` or `metadataOrigin`? Same elsewhere

Reply:
`metadataUri` makes sense to me